libs/execution_context/ebpf_maps.c

```diff
@@ -690,13 +688,13 @@ static ebpf_result_t
 _create_hash_map_internal(
     size_t map_struct_size,
     _In_ const ebpf_map_definition_in_memory_t* map_definition,
+    _In_ const size_t supplemental_value_size,
```

Comment:
```suggestion
    const size_t supplemental_value_size,
```
Don't use `_In_` with value types, only pointers.  Also probably don't need const here either.